internal/pkg/mount/switchroot/switchroot.go

```diff
@@ -98,6 +100,12 @@ func Switch(prefix string, mountpoints mount.Points) (err error) {
 		log.Printf("race detection enabled with halt_on_error=1")
 	}
 
+	if val := procfs.ProcCmdline().Get("talos.fips140"); val != nil && fipsmode.Enabled() {
+		if pointer.SafeDeref(val.First()) == "strict" {
+			envv = append(envv, fipsmode.StrictEnvironmentVariable())
```

Comment:
I guess `envv` is a previous typo

Reply:
nope, it's correct arg name here: https://pkg.go.dev/golang.org/x/sys/unix#Exec (goes from the C naming)